<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_principal</class>
 <widget class="QMainWindow" name="tela_principal">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Sex Shop</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/img/img/logo_sex.png</normaloff>:/img/img/logo_sex.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="fr_central">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_lateral">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #505050;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCalendarWidget" name="calendario">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>220</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QWidget
{ 
    background: qlineargradient(
    x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0.1 #F0F0F0, stop: 0.6 #A3A3A3);
}
QFrame {
    background-color:#FFFFFF;
	alternate-background-color: #D2D2D2;
}

QAbstractItemView {
	selection-background-color: rgb(255, 174, 0);
}
</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_hora">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>200</y>
         <width>220</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 211, 211);</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_data">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>180</y>
         <width>220</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 211, 211);</string>
       </property>
       <property name="text">
        <string>Ter√ßa-feira, 30 de outubro de 2012</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>18</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.300000000000000">
          <color alpha="255">
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </gradientstop>
         <gradientstop position="0.600000000000000">
          <color alpha="255">
           <red>82</red>
           <green>82</green>
           <blue>82</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QMenuBar {
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0.3 #777777, stop:0.6 #525252); 
 }

 QMenuBar::item {
     spacing: 3px; /* spacing between menu bar items */
     padding: 1px 4px;
     background: transparent;
	color: white;
 }

 QMenuBar::item:pressed {
     background: #888888;
 }

 QMenu::item:selected {
     background: rgba(100, 100, 100, 150);
 }</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="styleSheet">
     <string notr="true">background-color: #505050;
color: #FFFFFF;</string>
    </property>
    <property name="title">
     <string>Arquivo</string>
    </property>
    <widget class="QMenu" name="menuNovo">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #505050;
color: #FFFFFF;</string>
     </property>
     <property name="title">
      <string>Novo</string>
     </property>
     <addaction name="realizar_venda"/>
     <addaction name="Cadastrar_cliente"/>
     <addaction name="cadastrar_forncecedor"/>
     <addaction name="cadastrar_produto"/>
    </widget>
    <addaction name="menuNovo"/>
    <addaction name="botao_sair"/>
   </widget>
   <widget class="QMenu" name="menuListar">
    <property name="styleSheet">
     <string notr="true">background-color: #505050;
color: #FFFFFF;</string>
    </property>
    <property name="title">
     <string>Buscar</string>
    </property>
    <addaction name="buscar_clientes"/>
    <addaction name="buscar_produtos"/>
    <addaction name="buscar_fornecedores"/>
   </widget>
   <widget class="QMenu" name="menuConfigura_oes">
    <property name="styleSheet">
     <string notr="true"> background-color: #505050;
color: #FFFFFF;</string>
    </property>
    <property name="title">
     <string>Visualizar</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Cadastro_da_Loja"/>
    <addaction name="Estoque"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuListar"/>
   <addaction name="menuConfigura_oes"/>
  </widget>
  <widget class="QStatusBar" name="barra_de_status">
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #505050;
color: #FFFFFF;
font: 75 10pt &quot;Calibri&quot;;</string>
   </property>
  </widget>
  <widget class="QToolBar" name="barra_de_menus">
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Barra de menus</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #505050;
color: #FFFFFF;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="consultar_caixa"/>
   <addaction name="consultar_clientes"/>
   <addaction name="consultar_fornecedores"/>
   <addaction name="Estoque"/>
  </widget>
  <action name="consultar_clientes">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/busar_clientes.png</normaloff>:/img/img/busar_clientes.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Clientes</string>
   </property>
  </action>
  <action name="consultar_caixa">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/caixa.png</normaloff>:/img/img/caixa.png</iconset>
   </property>
   <property name="text">
    <string>Caixa</string>
   </property>
  </action>
  <action name="consultar_fornecedores">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/fornecedor_buscar.png</normaloff>:/img/img/fornecedor_buscar.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Forncedores</string>
   </property>
  </action>
  <action name="cadastrar_forncecedor">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/cadastrar_fornecedor.png</normaloff>:/img/img/cadastrar_fornecedor.png</iconset>
   </property>
   <property name="text">
    <string>Fornecedor</string>
   </property>
  </action>
  <action name="cadastrar_produto">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/cadastrar_produto.png</normaloff>:/img/img/cadastrar_produto.png</iconset>
   </property>
   <property name="text">
    <string>Produto</string>
   </property>
  </action>
  <action name="realizar_venda">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/vender.png</normaloff>:/img/img/vender.png</iconset>
   </property>
   <property name="text">
    <string>Venda</string>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="Cadastrar_cliente">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/adiciona_clientes.png</normaloff>:/img/img/adiciona_clientes.png</iconset>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="botao_sair">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/sair.png</normaloff>:/img/img/sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="buscar_clientes">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/busar_clientes.png</normaloff>:/img/img/busar_clientes.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Clientes</string>
   </property>
  </action>
  <action name="buscar_produtos">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/produto.png</normaloff>:/img/img/produto.png</iconset>
   </property>
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="buscar_fornecedores">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/fornecedor_buscar.png</normaloff>:/img/img/fornecedor_buscar.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Fornecedores</string>
   </property>
  </action>
  <action name="Cadastro_da_Loja">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/loja.png</normaloff>:/img/img/loja.png</iconset>
   </property>
   <property name="text">
    <string>Cadastro da Loja</string>
   </property>
  </action>
  <action name="Estoque">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/estoque.png</normaloff>:/img/img/estoque.png</iconset>
   </property>
   <property name="text">
    <string>Estoque</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
